app/helpers/format_helper.rb

```diff
@@ -102,13 +102,12 @@ def class_for_todo(bool)
     end
   end
 
-  # rubocop:disable Lint/EndAlignment
   def word_pluralize(count, singular, plural = nil)
     word = if (count == 1 || count =~ /^1(\.0+)?$/)
-      singular
-    else
-      plural || singular.pluralize
-    end
+             singular
+           else
```

Comment:
You are right, Rubocop only corrected line 107 and should correct 108-110. This is a Rubocop bug and according the releases notes, it is not yet fixed in the last Rubocop version. So I thing that it is nice to report bugs to Rubocop when you find then, and not only correct this manually. I've just an open an issue: https://github.com/bbatsov/rubocop/issues/4608